projects/common-platform-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/integrations/client/OsClient.kt

```diff
@@ -0,0 +1,130 @@
+package uk.gov.justice.digital.hmpps.integrations.client
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.service.annotation.GetExchange
+
+interface OsClient {
+    @GetExchange("/search/places/v1/find")
+    fun searchByFreeText(
+        @RequestParam query: String,
+        @RequestParam maxResults: Int,
+        @RequestParam key: String
```

Comment:
Do you still need to pass this as an explicit request parameter since you added the interceptor in the RestClientConfig?